plugins/dicom_viewer/web_client/views/DicomView.js

```diff
@@ -0,0 +1,349 @@
+        this.files = [];
+        this.index = 0;
+        this.first = true;
+        this.playing = false;
+        this.playRate = 500;
+        this.imageData = null;
+        this.imageDataCache = {};
+        this.tagCache = {};
+        this.xhr = null;
+        this.render();
+        this.loadFileList();
+    },
+
+    setIndex: function (index) {
+        if (index < 0 || index >= this.files.length) {
+            return;
+        }
+        this.index = index;
+        document.getElementById('dicom-slider').value = index;
+        this.loadFile(this.files[index]);
+    },
+
+    next: function () {
+        if (this.index === this.files.length - 1) {
+            this.setIndex(0);
+        } else {
+            this.setIndex(this.index + 1);
+        }
+    },
+
+    previous: function () {
+        if (this.index === 0) {
+            this.setIndex(this.files.length - 1);
+        } else {
+            this.setIndex(this.index - 1);
+        }
+    },
+
+    play: function () {
+        if (this.playing) {
+            this.playRate *= 0.5;
+            return;
+        }
+        this.playing = true;
+        this.step();
+    },
+
+    step: function () {
+        if (this.playing) {
+            this.next();
+            setTimeout(this.step.bind(this), this.playRate);
+        }
+    },
+
+    pause: function () {
+        this.playing = false;
+        this.playRate = 500;
+    },
+
+    loadFileList: function () {
+        restRequest({
+            path: '/item/' + this.item.get('_id') + '/dicom',
+            data: {
+                filters: 'dummy' // don't need the dicom tags, just want the sorted results
+            }
+        }).done(_.bind(function (resp) {
+            this.handleFileList(resp);
+        }, this));
+    },
+
+    handleFileList: function (files) {
+        this.files = files;
+        var slider = document.getElementById('dicom-slider');
+        slider.min = 0;
+        slider.max = this.files.length - 1;
+        slider.step = 1;
+        this.setIndex(0);
+    },
+
+    loadFile: function (file) {
+        if (file.name in this.imageDataCache) {
+            this.showCached(file);
+            return;
+        }
+        console.log(file.name);
+        if (this.xhr) {
+            this.xhr.abort();
+        }
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', apiRoot + '/file/' + file._id + '/download', true);
+        xhr.responseType = 'arraybuffer';
+        xhr.onload = _.bind(function (event) {
+            try {
+                const dataView = new DataView(xhr.response);
+                const image = daikon.Series.parseImage(dataView);
+                const imageData = createImageData(image);
+                this.imageDataCache[file.name] = imageData;
+                this.tagCache[file.name] = this.getTags(image);
+                this.showCached(file);
+            } catch (e) {
+                console.log(e);
+            }
+        }, this);
+        xhr.send();
```

Comment:
Hm, guess jqxhr doesn't support this natively. Might be a good idea to add another jQuery plugin into Girder core, following [this example](http://www.henryalgus.com/reading-binary-files-using-jquery-ajax/) so that we could have a binary ajaxTransport.